app/webpacker/controllers/side_menu_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "stimulus"
+
+export default class extends Controller {
+
+  changeActiveTab(event) {
+
+    this.dispatch("changeActiveComponent", { detail: { 
+        currentActiveTab: this.currentActiveTab,
+        newActiveTab: event.currentTarget
+      }})
+
+    this.currentActiveTab.classList.remove('selected')
+    event.currentTarget.classList.add('selected')
+  }
+
+  get currentActiveTab() {
+    return this.element.querySelector('.selected')
```

Comment:
As `.selected` is a very common class (selector), I'd vote to specify elements with target (let's say `tab` target)

Reply:
hmm yes, if we do that we can use `this.tabTargets.querySelector('.selected')` to get the active tab which is much more specific and intentional.

Reply:
This gave me the idea to use 
```js
    const currentActiveForm = this.formTargets.find(
      (form) => form.id == `${currentActiveTab.id}_form`
    );

    const newActiveForm = this.formTargets.find(
      (form) => form.id == `${newActiveTab.id}_form`
    );
```
in `enteprise_form_controller` :smile: 

And I've made the other changes you requested. Thanks jean!